frontend/ui/uimanager.lua

```diff
@@ -392,7 +392,7 @@ function UIManager:show(widget, refreshtype, refreshregion, x, y, refreshdither)
     for i = #self._window_stack, 0, -1 do
         local top_window = self._window_stack[i]
         -- skip modal window
-        if widget.modal or not top_window or not top_window.widget.modal then
+        if widget.modal or widget.toast or not top_window or not top_window.widget.modal then
```

Comment:
Shouldn't all toasts be modals to begin with?

Reply:
I guess so. But as we don't use/check widget.modal elsewhere it implies that a widget with .toast is handled just as a widget with .modal. So, I guess there's nothing else to do.